src/test/java/com/wikia/webdriver/elements/mercury/pages/ArticlePage.java

```diff
@@ -0,0 +1,45 @@
+package com.wikia.webdriver.elements.mercury.pages;
+
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.elements.common.Navigate;
+import com.wikia.webdriver.elements.mercury.components.Loading;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ArticlePage extends WikiBasePageObject {
+
+  @FindBy(css = "article a[href=\"/wiki/Category:Category_page_test"
```

Comment:
what about my first question? this selector is visible only on articles with this particular category, can we make this selector article content agnostic? so we can reuse it someday